test/integration.test.js

```diff
@@ -206,10 +206,10 @@ test.serial('Publish a release with release information in assets', async (t) =>
     .reply(200, {permissions: {push: true}})
     .post(`/repos/${owner}/${repo}/releases`, {
       tag_name: nextRelease.gitTag,
-      target_commitish: options.branch,
       name: nextRelease.gitTag,
       body: nextRelease.notes,
       draft: true,
+      prerelease: true,
```

Comment:
There is no `target_commitish` parameter for https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#upload-a-release-asset